crates/typst/src/layout/measure.rs

```diff
@@ -60,7 +73,12 @@ pub fn measure(
         None => context.styles().at(span)?,
     };
 
-    let pod = Regions::one(Axes::splat(Abs::inf()), Axes::splat(false));
+    let available = Axes::new(
+        width.resolve(context.styles().at(span)?).unwrap_or(Abs::inf()),
+        height.resolve(context.styles().at(span)?).unwrap_or(Abs::inf()),
```

Comment:
Why not just `styles` instead of `context.styles().at(span)?`?

Reply:
Good point. Didn’t notice that variable, so now I’ve changed the code to use it.